### ISSUE TITLE
Forums with lots of channels in conversation view

### ISSUE THREAD

**Issue by jamesn81** [esotalk/esoTalk#373]
When a forum has a lot of channels, their names take up a lot of space at the top of the view when choosing "Show the conversation list by default".

If possible, an option to hide all channels (and show only the current channel name) or at least truncate the list to some kind of fixed width (and not wrap them) would be great.

![screen shot 2014-10-27 at 6 48 31 pm](https://cloud.githubusercontent.com/assets/9331560/4788156/51c1cf14-5db6-11e4-9348-d55c69ef3608.png)

**Comment by tvb** [comment]
I would say that we only show the 4/5 most active channels and a link to 'show more channels' in a dropdown.

**Comment by jamesn81** [comment]
Yeah sure 4/5 - most active, or most recent and "show all" - perfect.

**Comment by ghost** [comment]
Well that's why there're sub-channels.

**Comment by tvb** [comment]
sub-channels does not justify all scenario's

**Comment by jamesn81** [comment]
@mashify - I need to limit each channel to a group/user in my scenario.
